contracts/src/v0.8/llo-feeds/test/reward-manager/RewardManager.updateRewardRecipients.t.sol

```diff
@@ -106,146 +106,6 @@ contract RewardManagerUpdateRewardRecipientsTest is BaseRewardManagerTest {
     updateRewardRecipients(PRIMARY_POOL_ID, recipients, ADMIN);
   }
 
-  function test_updateRecipientsToDifferentSet() public {
-    //create a list of containing recipients from the primary configured set, and new recipients
-    Common.AddressAndWeight[] memory recipients = new Common.AddressAndWeight[](getPrimaryRecipients().length + 4);
-    for (uint256 i; i < getPrimaryRecipients().length; i++) {
-      //copy the recipient and set the weight to 0 which implies the recipient is being replaced
-      recipients[i] = Common.AddressAndWeight(getPrimaryRecipients()[i].addr, 0);
-    }
-
-    //add the new recipients individually
-    recipients[4] = Common.AddressAndWeight(DEFAULT_RECIPIENT_5, ONE_PERCENT * 25);
-    recipients[5] = Common.AddressAndWeight(DEFAULT_RECIPIENT_6, ONE_PERCENT * 25);
-    recipients[6] = Common.AddressAndWeight(DEFAULT_RECIPIENT_7, ONE_PERCENT * 25);
-    recipients[7] = Common.AddressAndWeight(DEFAULT_RECIPIENT_8, ONE_PERCENT * 25);
-
-    //should revert as it's not possible to new recipients
-    vm.expectRevert(INVALID_ADDRESS_ERROR_SELECTOR);
-
-    //updating a recipient should force the funds to be paid out for the primary recipients
-    updateRewardRecipients(PRIMARY_POOL_ID, recipients, ADMIN);
```

Comment:
I think it might be worth checking if the new set was updated and the old set was removed